******************************************************************************
** Loading Vizard5.py
******************************************************************************
Running Depth Sensing sample ... Press 'Esc' to quit
Press 's' to save the point cloud
[Sample] Using default resolution
[2024-08-12 08:29:27 UTC][ZED][INFO] Logging level INFO
[2024-08-12 08:29:28 UTC][ZED][INFO] Using USB input... Switched to default resolution HD720
[2024-08-12 08:29:30 UTC][ZED][INFO] [Init]  Depth mode: ULTRA
[2024-08-12 08:29:31 UTC][ZED][INFO] [Init]  Camera successfully opened.
[2024-08-12 08:29:31 UTC][ZED][INFO] [Init]  Camera FW version: 1523
[2024-08-12 08:29:31 UTC][ZED][INFO] [Init]  Video mode: HD720@60
[2024-08-12 08:29:31 UTC][ZED][INFO] [Init]  Serial Number: S/N 39345765
Traceback (most recent call last):
  File "<string>", line 10, in <module>
  File "C:\Users\12016\OneDrive\Documents\Vizard\Camelot\Vizard5.py", line 103, in <module>
    main()
  File "C:\Users\12016\OneDrive\Documents\Vizard\Camelot\Vizard5.py", line 72, in main
    viewer.init(1,sys.argv,camera_model,res)
  File "C:\Users\12016\OneDrive\Documents\Vizard\Camelot\ogl_viewer\viewer.py", line 229, in init
    glutInit(_argc, _argv)
  File "C:\Program Files\WorldViz\Vizard7\bin\lib\site-packages\OpenGL\GLUT\special.py", line 333, in glutInit
    _base_glutInit( ctypes.byref(count), holder )
  File "C:\Program Files\WorldViz\Vizard7\bin\lib\site-packages\OpenGL\platform\baseplatform.py", line 423, in __call__
    raise error.NullFunctionError(
OpenGL.error.NullFunctionError: Attempt to call an undefined function glutInit, check for bool(glutInit) before calling
** Load Time: 6.11 seconds
